---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "Genome-wide Analysis of Multi-View Data of miRNA-seq to Identify miRNA Biomarkers for Stomach Cancer"
authors: ["Namrata Pant", "admin", "Sushmita Paul", "Indrajit Saha"]
date: 2019-07-25T20:40:46+05:30
doi: "10.1016/j.jbi.2019.103254"

# Schedule page publish date (NOT publication's date).
publishDate: 2019-07-25T20:40:46+05:30

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Journal of Biomedical Informatics"
publication_short: "JBI"

abstract: "Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis."

# Summary. An optional shortened abstract.
summary: "Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis."

tags: ["computational biology"]
categories: []
featured: true

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf: https://doi.org/10.1016/j.jbi.2019.103254
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: "Line chart showing the distribution of weights for ranks of the total number
of miRNAs in our dataset"
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
