+++
title = "Identification of Four miRNAs by Analysing Multi-View miRNA-seq Data of Stomach Cancer"
date = 2019-04-20T11:01:12+02:00

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["admin", "Namrata Pant", "Sushmita Paul", "Indrajit Saha"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Preprint / Working Paper
# 4 = Report
# 5 = Book
# 6 = Book section
# 7 = Thesis
# 8 = Patent
publication_types = ["1"]

# Publication name and optional abbreviated version.
publication = "IEEE TENSYMP 2019"
publication_short = "TENSYMP 2019"

# Abstract.
abstract = "Stomach cancer continues to be one of the most common cancer types in the world. Poor prognosis and late detection are major challenges in the diagnosis and treatment of this cancer. Currently, Next-Generation Sequencing (NGS) of genome has revolutionized stomach cancer research by providing multi-view data which can be used for better understanding of the underlying molecular mechanism of this disease. Micro-RNA (miRNA) sequencing data is one such high resolution expression data. MiRNAs are evolutionary conserved, small, non-coding RNAs that play a role in post-transcriptional regulation of gene expression. They are proven to show abnormal expression for a specific biological condition like tumor or age or survival of stomach cancer which makes them potential biomarkers for this cancer type. MiRNA data analysis comes with a challenge of less number of samples as compared to the number of miRNAs. Finding a small set of miRNAs is needed to identify potential biomarkers. In this regard, here, a method for selecting a small subset of miRNAs from the entire miRNA expression data has been proposed that selects miRNAs, common to three different categories of clinical outcomes, viz. condition, age, and survival status. First, three feature selection methods have been used to rank miRNAs individually for different categories. These ranks are then used to compute an ensemble of ranks of each miRNA using adaptive weight method for each category. Second, the top 100 miRNAs from each category have been used to find the miRNAs that are common to all categories. As a result, four miRNAs are found which are validated using classification of subclass under each category, miRNA-Gene-TF network, PPI network, expression analysis using box plots, KEGG pathway and GO enrichment analysis."

# Summary. An optional shortened abstract.
summary = ""

# Digital Object Identifier (DOI)
doi = ""

# Is this a featured publication? (true/false)
featured = true

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["computational-biology", "python"]

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = []

# Slides (optional).
#   Associate this page with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Links (optional).
url_pdf = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# links = [{name = "Custom Link", url = "http://example.org"}]

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
