<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.5.0"><meta name=author content="Somnath Rakshit"><meta name=description content="In this work, it is seen that deep learning technique such as stacked autoencoder can be useful in reducing the dimension of a high dimensional multi-omics data. The performance obtained using this architecture is better than the other methods as shown here. Besides, some genes are identified which are in the pathways of breast cancer, as determined by the deep selection of features and the Bonferroni correction of p-values of the result of the one-sample t-test."><link rel=alternate hreflang=en-us href=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/><meta name=theme-color content=#2962ff><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.6.0/css/all.css integrity=sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/academic.min.04a87a1cb9027e3c50f566322527c56f.css><script>window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;ga('create','UA-107696056-2','auto');ga('require','eventTracker');ga('require','outboundLinkTracker');ga('require','urlChangeTracker');ga('send','pageview');</script><script async src=https://www.google-analytics.com/analytics.js></script><script async src=https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin=anonymous></script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/img/icon-32.png><link rel=apple-touch-icon type=image/png href=/img/icon-192.png><link rel=canonical href=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/><meta property=twitter:card content=summary><meta property=twitter:site content=@_SomnathRakshit><meta property=twitter:creator content=@_SomnathRakshit><meta property=og:site_name content="Somnath Rakshit"><meta property=og:url content=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/><meta property=og:title content="Deep Learning for Integrated Analysis of Breast Cancer Subtype Specific Multi-omics Data | Somnath Rakshit"><meta property=og:description content="In this work, it is seen that deep learning technique such as stacked autoencoder can be useful in reducing the dimension of a high dimensional multi-omics data. The performance obtained using this architecture is better than the other methods as shown here. Besides, some genes are identified which are in the pathways of breast cancer, as determined by the deep selection of features and the Bonferroni correction of p-values of the result of the one-sample t-test."><meta property=og:image content=https://somnathrakshit.github.io/img/icon-192.png><meta property=twitter:image content=https://somnathrakshit.github.io/img/icon-192.png><meta property=og:locale content=en-us><meta property=article:published_time content=2018-10-31T01:23:15&#43;02:00><meta property=article:modified_time content=2018-10-31T01:23:15&#43;02:00><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/"},"headline":"Deep Learning for Integrated Analysis of Breast Cancer Subtype Specific Multi-omics Data","datePublished":"2018-10-31T01:23:15+02:00","dateModified":"2018-10-31T01:23:15+02:00","author":{"@type":"Person","name":"Somnath Rakshit"},"publisher":{"@type":"Organization","name":"Somnath Rakshit","logo":{"@type":"ImageObject","url":"https://somnathrakshit.github.io/img/icon-512.png"}},"description":"In this work, it is seen that deep learning technique such as stacked autoencoder can be useful in reducing the dimension of a high dimensional multi-omics data. The performance obtained using this architecture is better than the other methods as shown here. Besides, some genes are identified which are in the pathways of breast cancer, as determined by the deep selection of features and the Bonferroni correction of p-values of the result of the one-sample t-test."}</script><title>Deep Learning for Integrated Analysis of Breast Cancer Subtype Specific Multi-omics Data | Somnath Rakshit</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id=navbar-main><div class=container><a class=navbar-brand href=/>Somnath Rakshit</a>
<button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="collapse navbar-collapse" id=navbar><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/project><span>Projects</span></a></li><li class=nav-item><a class="nav-link  active" href=/publication><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/post><span>Blog</span></a></li><li class=nav-item><a class=nav-link href=/files/Somnath_Rakshit_Resume.pdf><span>Resume</span></a></li></ul><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-dark-toggle" href=#><i class="fas fa-moon" aria-hidden=true></i></a></li></ul></div></div></nav><div class=pub><div class="article-container pt-3"><h1>Deep Learning for Integrated Analysis of Breast Cancer Subtype Specific Multi-omics Data</h1><div class=article-metadata><div><span><a href=/authors/admin/>Somnath Rakshit</a></span>, <span><a href=/authors/subha-shankar-chakraborty/>Subha Shankar Chakraborty</a></span>, <span><a href=/authors/indrajit-saha/>Indrajit Saha</a></span>, <span><a href=/authors/dariusz-plewczynski/>Dariusz Plewczynski</a></span></div><span class=article-date>October 2018</span><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/&amp;text=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/&amp;t=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook-f"></i></a></li><li><a href="mailto:?subject=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data&amp;body=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/&amp;title=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="https://web.whatsapp.com/send?text=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data%20https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https://somnathrakshit.github.io/publication/breast-cancer-integrated-sae/&amp;title=Deep%20Learning%20for%20Integrated%20Analysis%20of%20Breast%20Cancer%20Subtype%20Specific%20Multi-omics%20Data" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div></div><div class="btn-links mb-3"><button type=button class="btn btn-outline-primary my-1 mr-1 js-cite-modal" data-filename=/publication/breast-cancer-integrated-sae/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1" href=http://www.nitttrkol.ac.in/indrajit/projects/integrated-analysis-breastcancer-subtypes/ target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1" href=http://www.nitttrkol.ac.in/indrajit/projects/integrated-analysis-breastcancer-subtypes/ target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1" href=/files/download/breast_cancer_sae_slides.pdf target=_blank rel=noopener>Slides</a>
<a class="btn btn-outline-primary my-1 mr-1" href=https://doi.org/10.1109/TENCON.2018.8650144 target=_blank rel=noopener>DOI</a></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Breast cancer is a deadly disease which commonly occurs all over the world and has been found to be the largest cause of cancer in females. Its detection is still a major challenge, both from a computational as well as from a biological point of view. Next Generation Sequencing (NGS) techniques have accelerated the mapping of human genomes rapidly. Involvement of advanced NGS techniques reveal that multiple genetic components are responsible for the cause of breast cancer and its subtypes. However, the high volume of data that is produced by the NGS techniques are difficult to study because of their high dimensionality and complexity. Thus, the integrated study of multi-omics data is one of the major challenges in medical science. This fact motivated us to study the NGS based high throughput expression data of miRNAs and mRNAs as well as Beta values of DNA methylation of the corresponding mRNAs. In this regard, first, these datasets, together consisting of 33564 features of 305 patients in five classes viz. Luminal A, Luminal B, HER2-enriched, Basal-like and Control, are analysed in an integrated fashion using deep learning technique to classify the breast cancer subtypes properly. Second, the results of the deep learning technique are further analysed in order to identify the deeply connected features, i.e. either miRNA or mRNA or DNA Methylation, which are pivotal in the classification of breast cancer subtypes as well as play a crucial role in the formation of breast cancer. For this purpose, a deep learning technique, called stacked autoencoder is used to transform the features into a low dimensional space, which is then fed to the five well known classifiers for classification. Moreover, the same transformed data is used to select the potential features after performing multiplication with the original data and Bonferroni correction on the p-values produced by the one-sample t-test. The results have been validated quantitatively and through biological significance analysis. During the biological analysis, it is observed that TP53 and BRCA1 have been found, which is known to play a crucial role in breast cancer formation.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#1>Conference paper</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9">IEEE TENCON</div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style></div><div class=article-tags><a class="badge badge-light" href=/tags/deep-learning/>deep learning</a>
<a class="badge badge-light" href=/tags/computational-biology/>computational biology</a></div><div class="media author-card"><img class="portrait mr-3" src=/authors/admin/avatar_hu19f97f82492f8a6a0c19e0126018b235_339080_250x250_fill_q90_lanczos_center.jpg alt=Avatar><div class=media-body><h5 class=card-title><a href=https://somnathrakshit.github.io/>Somnath Rakshit</a></h5><h6 class=card-subtitle>Masters Student (Data Science Track)</h6><p class=card-text>I am a Masters student (Data Science track) at the School of Information, The University of Texas at Austin. My current interests lie in deep learning, natural language processing and knowledge graphs.</p><ul class=network-icon aria-hidden=true><li><a href=https://www.linkedin.com/in/somnathrakshit target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href=https://twitter.com/_SomnathRakshit target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href="https://scholar.google.co.in/citations?user=9UaFRzYAAAAJ&amp;hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li><li><a href=https://github.com/somnathrakshit target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href=/files/Somnath_Rakshit_Resume.pdf><i class="ai ai-cv"></i></a></li></ul></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/python.min.js></script><script>hljs.initHighlightingOnLoad();</script><script>const search_index_filename="/index.json";const i18n={'placeholder':"Search...",'results':"results found",'no_results':"No results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/academic.min.130521ecfc6f534c52c158217bbff718.js></script><div class=container><footer class=site-footer><p class=powered-by>&copy; Somnath Rakshit, 2019 &middot; .
<span class=float-right aria-hidden=true><a href=# id=back_to_top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&times;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>