<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.7.0"><meta name=author content="Somnath Rakshit"><meta name=description content="Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis."><link rel=alternate hreflang=en-us href=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css integrity="sha256-4w9DunooKSr3MFXHXWyFER38WmPdm361bQS/2KUWZbU=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/academic.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-107696056-2"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-107696056-2',{});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu8f513581b82a7ea9f3a68e0c5d348cec_18530_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu8f513581b82a7ea9f3a68e0c5d348cec_18530_192x192_fill_lanczos_center_2.png><link rel=canonical href=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@_SomnathRakshit"><meta property="twitter:creator" content="@_SomnathRakshit"><meta property="og:site_name" content="Somnath Rakshit"><meta property="og:url" content="https://somnathrakshit.github.io/publication/multi-view-mirna-stad/"><meta property="og:title" content="Genome-wide Analysis of Multi-View Data of miRNA-seq to Identify miRNA Biomarkers for Stomach Cancer | Somnath Rakshit"><meta property="og:description" content="Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis."><meta property="og:image" content="https://somnathrakshit.github.io/publication/multi-view-mirna-stad/featured.jpg"><meta property="twitter:image" content="https://somnathrakshit.github.io/publication/multi-view-mirna-stad/featured.jpg"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2019-07-25T20:40:46+05:30"><meta property="article:modified_time" content="2019-07-25T20:40:46+05:30"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://somnathrakshit.github.io/publication/multi-view-mirna-stad/"},"headline":"Genome-wide Analysis of Multi-View Data of miRNA-seq to Identify miRNA Biomarkers for Stomach Cancer","image":["https://somnathrakshit.github.io/publication/multi-view-mirna-stad/featured.jpg"],"datePublished":"2019-07-25T20:40:46+05:30","dateModified":"2019-07-25T20:40:46+05:30","author":{"@type":"Person","name":"Namrata Pant"},"publisher":{"@type":"Organization","name":"Somnath Rakshit","logo":{"@type":"ImageObject","url":"https://somnathrakshit.github.io/images/icon_hu8f513581b82a7ea9f3a68e0c5d348cec_18530_192x192_fill_lanczos_center_2.png"}},"description":"Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis."}</script><title>Genome-wide Analysis of Multi-View Data of miRNA-seq to Identify miRNA Biomarkers for Stomach Cancer | Somnath Rakshit</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Somnath Rakshit</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Somnath Rakshit</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/project><span>Projects</span></a></li><li class=nav-item><a class="nav-link active" href=/publication><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/post><span>Blog</span></a></li><li class=nav-item><a class=nav-link href=/files/Somnath_Rakshit_Resume.pdf><span>Resume</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-dark-toggle" href=#><i class="fas fa-moon" aria-hidden=true></i></a></li></ul></div></nav><div class=pub><div class="article-container pt-3"><h1>Genome-wide Analysis of Multi-View Data of miRNA-seq to Identify miRNA Biomarkers for Stomach Cancer</h1><div class=article-metadata><div><span>Namrata Pant</span>, <span>Somnath Rakshit</span>, <span>Sushmita Paul</span>, <span>Indrajit Saha</span></div><span class=article-date>July 2019</span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary my-1 mr-1" href=https://doi.org/10.1016/j.jbi.2019.103254 target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 js-cite-modal" data-filename=/publication/multi-view-mirna-stad/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1" href=https://doi.org/10.1016/j.jbi.2019.103254 target=_blank rel=noopener>DOI</a></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:720px;max-height:539px><div style=position:relative><img src=/publication/multi-view-mirna-stad/featured_hub099b7059c0f19b5068497a08653ace0_47319_720x0_resize_q90_lanczos.jpg alt class=featured-image>
<span class=article-header-caption>Line chart showing the distribution of weights for ranks of the total number of miRNAs in our dataset</span></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Stomach cancer is one of the leading causes of cancer-related deaths worldwide. More than 80% diagnosis of this cancer occur at later stages leading to low 5-year survival rate. This emphasizes the need to have better prognostic techniques for stomach cancer. In this regard, the Next-Generation Sequencing of whole genome and multi-view approach to omics may reveal the underlying molecular complexity of stomach cancer using high throughput expression data of miRNA. Generally, miRNAs are small, non-coding RNAs, which cause downregulation of target mRNAs. They also show differential expression for a specific biological condition like stage or histological type of stomach cancer, highlighting their importance as potential biomarkers. Analyzing miRNA expression data is a challenging task due to the existence of large number of miRNAs and less sample size. A small set of miRNAs will be helpful in designing efficient diagnostic and prognostic tool. In this regard, here a computational framework is proposed that selects different sets of miRNAs for five different categories of clinical outcomes viz. condition, clinical stage, age, histological type, and survival status. First, the miRNAs are ranked using four feature ranking methods. These ranks are used to find an ensemble rank based on adaptive weight. Second, the top 100 miRNAs from each category are used to find the miRNAs that are common to all categories as well as miRNAs that belong to only one category. Finally, the results have been validated quantitatively and through biological significance analysis.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9">Journal of Biomedical Informatics</div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style></div><div class=article-tags><a class="badge badge-light" href=/tags/computational-biology/>computational biology</a></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/&text=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/&t=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer&body=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/&title=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="https://web.whatsapp.com/send?text=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer%20https://somnathrakshit.github.io/publication/multi-view-mirna-stad/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https://somnathrakshit.github.io/publication/multi-view-mirna-stad/&title=Genome-wide%20Analysis%20of%20Multi-View%20Data%20of%20miRNA-seq%20to%20Identify%20miRNA%20Biomarkers%20for%20Stomach%20Cancer" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><div class=media-body><h5 class=card-title><a href=/authors/namrata-pant/></a></h5><ul class=network-icon aria-hidden=true></ul></div></div><div class="article-widget content-widget-hr"><h3>Related</h3><ul><li><a href=/publication/breast-cancer-integrated-sae/>Deep Learning for Integrated Analysis of Breast Cancer Subtype Specific Multi-omics Data</a></li></ul></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/highlight.min.js integrity="sha256-1zu+3BnLYV9LdiY85uXMzii3bdrkelyp37e0ZyTAQh0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/languages/python.min.js></script><script>const code_highlighting=true;</script><script>const isSiteThemeDark=false;</script><script>const search_config={"indexURI":"/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/js/academic.min.a8d7005002cb4a052fd6d721e83df9ba.js></script><div class=container><footer class=site-footer><p class=powered-by>Â© Somnath Rakshit, 2020 &#183; .
<span class=float-right aria-hidden=true><a href=# id=back_to_top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>