---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: "Identification of Epigenetic Biomarkers with the use of Gene Expression and DNA Methylation for Breast Cancer Subtypes"
authors: ["Indrajit Saha", "admin", "Michal Wlasnowolski", "Dariusz Plewczynski"]
date: 2019-08-02T20:02:51+05:30
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: 2019-08-02T20:02:51+05:30

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "IEEE TENCON 2019"
publication_short: "TENCON 2019"

abstract: "Breast cancer is one of the most deadly cancers. It has four subtypes: Luminal A (LA), Luminal B (LB),  HER2-enriched (HER2-E) and Basal-like (BL). For the cause of breast cancer subtypes, there are different genetic and epigenetic factors involved in its progression and susceptibility. Thus, the identification of genetic and/or epigenetic biomarkers can be helpful to understand the biological mechanisms better and to improve the diagnostic processes of this disease and its subtypes. Hence, this fact motivated us to investigate the epigenetic factor, such as DNA Methylation, with the integration of gene expression in order to find epigenetic biomarkers for breast cancer subtypes. In this regard, first, we identified set of up and down regulated genes for each subtype using differential analysis. Thereafter, regression based feature ranking problem is formed in order to find the DNA Methylation site that is mostly responsible for the change in expression of a gene, which is considered as an epigenetic biomarker. A bagging integrated ensemble of decision trees is used for the same. The results of top ten up and down regulated genes and their corresponding most significant DNA Methylation sites are reported for breast cancer subtypes. Moreover, these genes are validated visually by means of survival and expression plots, showing TF-Gene-DNA Methylation interactions, Protein-Protein interaction network, KEGG pathway and GO enrichment analysis. The results show that top differentially expressed up and down regulated genes viz. MMP11, NUF2, EXO1, HJURP,  HOXA4, SYNM, CAV1 and COL4A3BP in breast cancer subtypes may change their expression because of DNA Methylation sites viz. cg22418565, cg26029744, cg24741598, cg04550103, cg25952581, cg02109162, cg18498156 and cg04985097 respectively."

# Summary. An optional shortened abstract.
summary: ""

tags: ["computational biology"]
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
